lib/Lift/CodeGenerator.cpp

```diff
@@ -1332,7 +1221,6 @@ void CodeGenerator::translate(Optional<uint64_t> RawVirtualAddress) {
   PostInstCombinePM.run(*TheModule);
 
   // Serialize an empty Model into TheModule
-  TupleTree<model::Binary> Model;
 
```

Comment:
Drop the comment too

Reply:
not sure what is going on here, my file has only 1294 lines

Reply:
a found it